.travis.yml

```diff
@@ -144,24 +144,17 @@ jobs:
 
     - stage: Build
       before_script: *auto_skip
-      name: 11 on alpine
+      name: 10 on buster
```

Comment:
These and the following changes to add the buster variant should also be in the `travis.yml.template` file which is the source template for the `.travis.yml` file.